src/Sylius/Behat/Client/ApiPlatformClient.php

```diff
@@ -153,6 +163,14 @@ public function responseHasValue(string $key, $value): bool
         return $this->getResponseContentValue($key) === $value;
     }
 
+    /** @param string|int $value */
+    public function relatedResourceHasValue(string $resource, string $key, $value): bool
+    {
+        $this->showRelated($resource);
+
+        return $this->getResponseContentValue($key) === $value;
+    }
```

Comment:
Wrong order , I guess this method should be above `public function responseHasValue(string $key, $value): bool`